manifest.yml

```diff
@@ -1,12 +1,10 @@
 manifest_version: 1
-name: ImageNormalizer
-kind: crafter
-type: pod
-description: '`ImageNormalizer` resizes and normalizes images'
+name: FlairTextEncoder
+kind: encoder
+description: ImagePaddlehubEncoder encodes text data.
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
-documentation: https://github.com/jina-ai/jina-hub
 version: 0.0.1
 license: apache-2.0
-keywords: [image, normalization, reader, resize]
```

Comment:
I changed the keywords of nlp to text, and torch to PyTorch to conform with our 2.0 executor guideline

Reply:
thanks a lot!